Source/ModelGroup.cs

```diff
@@ -140,6 +140,481 @@ public static List<Runtime.MeshPrimitive> CreateMultiPrimitivePlane()
             }
         }
 
+        protected static class Gltf
+        {
+            public static Runtime.GLTF CreateMultiNode()
+            {
+                List<Vector3> vertexPositions = new List<Vector3>()
```

Comment:
I would recommend putting this data in a separate file

Reply:
Fixed